docs/accessibility/get-started/introduction.mdx

```diff
@@ -6,22 +6,47 @@ sidebar_position: 10
+Dive deep into each violation with live, fully-rendered DOM snapshots of your application as it appeared during your tests.
+Track your team's progress over time with historical scores to monitor improvements and identify regressions.
+Looking for a general guide for Accessibility testing in Cypress App? [Go here.](/app/guides/accessibility-testing)
 
-## What is Cypress Accessibility?
-
-Cypress Accessibility provides organized, visual, and actionable accessibility reports, based completely on the tests you record to Cypress Cloud, and powered by Axe Core® by Deque Systems. This is the documentation page for that product. To learn about the general topic of accessibility testing using the Cypress App, see [our guide on this topic](/app/guides/accessibility-testing).
-
-Cypress Accessibility generates a sortable, filterable report, to see scores and violation counts by page/component, or a combined rule-based report for the entire run. It also includes a drill-in view with live DOM snapshots highlighting each issue detected.
+<Btn
+  label="Request trial ➜"
+  variant="purple-dark"
+  href="https://www.cypress.io/accessibility?utm_medium=intro-cta&utm_source=docs.cypress.io&utm_content=Request%20trial"
+/>
+<Btn
+  label="See a demo"
+  icon="action-play-small"
+  href="https://www.loom.com/share/eedb1d18c22942d39a5ed2471ae72788?sid=e39f791e-c0b3-4efc-9770-cbfac403c3b9"
+/>
+<Btn
+  label=" Explore an example project"
+  icon="cypress-accessibility-outline"
+  href="https://on.cypress.io/rwa-accessibility-views?utm_source=docs.cypress.io&utm_medium=intro-cta&utm_content=Explore%20an%20example%20project"
+/>
 
 <DocsImage
   src="/img/accessibility/get-started/cypress-accessibility-overview.png"
   alt="A graphic showing the Cypress Accessibility dashboard including list of rules, views, and the user interface showing how devtools can be used"
+  noBorder={true}
 />
 
-## How are reports created?
+## Get Started
+
+You don't need to make any changes to your code or your test setup. If you record test runs to the Cypress Cloud with Test Replay, you're ready to start using Cypress Accessibility.
+You'll start your [free trial](https://www.cypress.io/accessibility?utm_medium=intro-cta&utm_source=docs.cypress.io&utm_content=Request%20trial) with personalized support
+from our sales team and your existing test data to instantly see full accessibility reports for your all Cypress projects.
+From there, you can integrate with CI to set your own standards for handling the results.
+
+## How it Works
+
+Cypress Accessibility provides organized, visual, and actionable accessibility reports, based completely on the tests you record to Cypress Cloud, and powered by Axe Core® by Deque Systems. This is the documentation page for that product. To learn about the general topic of accessibility testing using the Cypress App, see [our guide on this topic](/app/guides/accessibility-testing).
```

Comment:
> This is the documentation page for that product. 

Confusing statement. What is the product? "Cypress Accessibility" or "Axe Core".

Perhaps this sentence could be removed?

Reply:
Yah we’re going to change all of this content actually, it’s the old content I left on the bottom